dashboard/src/api/resources/Resources.js

```diff
@@ -11,19 +11,6 @@ export const resourceTypes = Object.freeze({
   PRIMARY_DATA: "Primary Data",
 });
 
-export const resourceVersions = Object.freeze({
```

Comment:
Make a class called Resource, if you don't already have one. Make all these maps methods of that class. Examples:
isVersioned, Path, etc.